Traceback (most recent call last):
  File "/Users/dsc/miniforge3/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/dsc/miniforge3/lib/python3.9/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/dsc/miniforge3/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/dsc/miniforge3/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Users/dsc/miniforge3/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/Users/dsc/miniforge3/lib/python3.9/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Users/dsc/miniforge3/lib/python3.9/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/dsc/miniforge3/lib/python3.9/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
txt=input("Get some text:")
print(txt)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mStdinNotImplementedError[0m                  Traceback (most recent call last)
Input [0;32mIn [1][0m, in [0;36m<cell line: 1>[0;34m()[0m
[0;32m----> 1[0m txt[38;5;241m=[39m[38;5;28;43minput[39;49m[43m([49m[38;5;124;43m"[39;49m[38;5;124;43mGet some text:[39;49m[38;5;124;43m"[39;49m[43m)[49m
[1;32m      2[0m [38;5;28mprint[39m(txt)

File [0;32m~/miniforge3/lib/python3.9/site-packages/ipykernel/kernelbase.py:1158[0m, in [0;36mKernel.raw_input[0;34m(self, prompt)[0m
[1;32m   1151[0m [38;5;124;03m"""Forward raw_input to frontends[39;00m
[1;32m   1152[0m 
[1;32m   1153[0m [38;5;124;03mRaises[39;00m
[1;32m   1154[0m [38;5;124;03m------[39;00m
[1;32m   1155[0m [38;5;124;03mStdinNotImplementedError if active frontend doesn't support stdin.[39;00m
[1;32m   1156[0m [38;5;124;03m"""[39;00m
[1;32m   1157[0m [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m [38;5;28mself[39m[38;5;241m.[39m_allow_stdin:
[0;32m-> 1158[0m     [38;5;28;01mraise[39;00m StdinNotImplementedError(
[1;32m   1159[0m         [38;5;124m"[39m[38;5;124mraw_input was called, but this frontend does not support input requests.[39m[38;5;124m"[39m
[1;32m   1160[0m     )
[1;32m   1161[0m [38;5;28;01mreturn[39;00m [38;5;28mself[39m[38;5;241m.[39m_input_request(
[1;32m   1162[0m     [38;5;28mstr[39m(prompt),
[1;32m   1163[0m     [38;5;28mself[39m[38;5;241m.[39m_parent_ident[[38;5;124m"[39m[38;5;124mshell[39m[38;5;124m"[39m],
[1;32m   1164[0m     [38;5;28mself[39m[38;5;241m.[39mget_parent([38;5;124m"[39m[38;5;124mshell[39m[38;5;124m"[39m),
[1;32m   1165[0m     password[38;5;241m=[39m[38;5;28;01mFalse[39;00m,
[1;32m   1166[0m )

[0;31mStdinNotImplementedError[0m: raw_input was called, but this frontend does not support input requests.
StdinNotImplementedError: raw_input was called, but this frontend does not support input requests.

